src/components/nodes/components/NodesTableHead.vue

```diff
@@ -2,7 +2,9 @@
   <thead :class="classes.root">
     <tr>
       <th :class="classes.checkbox" v-if="!hideCheckbox" />
-      <th :class="classes.label" class="pl-0 pr-2" v-if="!hideLabel">{{ t('nodes.coin') }}</th>
+      <th :class="classes.label" class="pl-0 pr-2" v-if="!hideLabel">
+        {{ customLabelCode ? t(customLabelCode) : t('nodes.coin') }}
+      </th>
```

Comment:
```suggestion
      <th :class="classes.label" class="pl-0 pr-2" v-if="!hideLabel">
        {{ label }}
      </th>
```

1. Rename prop to `label`
2. Pass it already translated, e.g. `<NodesTableHead :label="$t('nodes.service')" />`
3. Pass the label explicitly for "ADM" and "Coins tab", e.g. `<NodesTableHead :label="$t('nodes.coin')" />`